src/rpc/blockchain.cpp

```diff
@@ -2635,7 +2649,9 @@ UniValue CreateUTXOSnapshot(NodeContext& node, CChainState& chainstate, CAutoFil
     result.pushKV("coins_written", stats.coins_count);
     result.pushKV("base_hash", tip->GetBlockHash().ToString());
     result.pushKV("base_height", tip->nHeight);
-
+    result.pushKV("path", path.u8string());
+    result.pushKV("txoutset_hash", stats.hashSerialized.ToString());
+    result.pushKV("nchaintx", static_cast<int64_t>(tip->nChainTx));
```

Comment:
Can you explain why a cast is needed? If one is needed, it might be better to use a non-narrowing.

```suggestion
    result.pushKV("nchaintx", int64_t{tip->nChainTx});
```

Reply:
```
rpc/blockchain.cpp:2654:12: error: call to member function 'pushKV' is ambiguous
    result.pushKV("nchaintx", tip->nChainTx);
    ~~~~~~~^~~~~~
./univalue/include/univalue.h:125:10: note: candidate function
    bool pushKV(const std::string& key, int64_t val_) {
         ^
./univalue/include/univalue.h:129:10: note: candidate function
    bool pushKV(const std::string& key, uint64_t val_) {
         ^
./univalue/include/univalue.h:133:10: note: candidate function
    bool pushKV(const std::string& key, bool val_) {
         ^
./univalue/include/univalue.h:137:10: note: candidate function
    bool pushKV(const std::string& key, int val_) {
         ^
./univalue/include/univalue.h:141:10: note: candidate function
    bool pushKV(const std::string& key, double val_) {
         ^
1 error generated.
```

I'll change the cast as you specify.